shared/contracts/BaseTemplate.sol

```diff
@@ -128,6 +132,25 @@ contract BaseTemplate is APMNamehash, IsContract {
         _acl.createPermission(_grantee, _agent, _agent.RUN_SCRIPT_ROLE(), _manager);
     }
 
+    /* PAYROLL */
+
+    function _installPayrollApp(Kernel _dao, Finance _finance, address _denominationToken, IFeed _priceFeed, uint64 _rateExpiryTime) internal returns (Payroll) {
+        Payroll payroll = Payroll(_installNonDefaultApp(_dao, PAYROLL_APP_ID));
+        payroll.initialize(_finance, _denominationToken, _priceFeed, _rateExpiryTime);
+        return payroll;
+    }
+
+    function _createPayrollPermissions(ACL _acl, Payroll _payroll, address _grantee, address _manager) internal {
+        _acl.createPermission(_grantee, _payroll, _payroll.ADD_BONUS_ROLE(), _manager);
+        _acl.createPermission(_grantee, _payroll, _payroll.ADD_EMPLOYEE_ROLE(), _manager);
+        _acl.createPermission(_grantee, _payroll, _payroll.ADD_REIMBURSEMENT_ROLE(), _manager);
```

Comment:
We may want a different manager for the `ADD_BONUS`, `ADD_REIMBURSEMENT`, and `ADD_EMPLOYEE` roles.

The other ones are very powerful “setting” management roles, whereas these are directly targeted at managing employees. Perhaps we can have the concept of the “Payroll settings manager” (should be Voting or etc.) and the “Payroll employee manager” (could be a high privilege account)?